### PR TITLE
Recreate np's consent doc PR

### PR DESCRIPTION
**Pull request by jhaaaa**
Recreate https://github.com/xmtp/docs-xmtp-org/pull/91

### CODE REVIEW

File: docs/pages/inboxes/user-consent/support-user-consent.md [hunk 1]
```diff
@@ -132,38 +136,183 @@ try await conversation.updateConsent(.allowed) // .allowed | .denied
...
+// Get consent directly on the member
+let memberConsentStates = try await group.members.map(\.consentState)
+
+// Get consent from the inboxId
+let inboxConsentState = try await client.preferences.inboxIdState(inboxId: inboxId)
+```
+
+:::
+
+## See who created and added you to a group
```
Comment (jhaaaa) [inline docs/pages/inboxes/user-consent/support-user-consent.md:288]: Not sure if this suggested edit is correct:

```suggestion
## See who created a group and added a user to it
```

Comment (nplasterer) [reply docs/pages/inboxes/user-consent/support-user-consent.md:288]: The method is `group.addedByInboxId` so it really only tells you who added you to the group. So you can't see who added someone else with this method.

Comment (nplasterer) [reply docs/pages/inboxes/user-consent/support-user-consent.md:288]: So as long as thats clear I'm fine with the edit.

**Review by nplasterer — APPROVED** [review]

File: docs/pages/inboxes/user-consent/support-user-consent.md (hunk 1 above)
Comment (jhaaaa) [reply docs/pages/inboxes/user-consent/support-user-consent.md:288]: Ah, got it - your original conveys this detail more clearly!

**Pull request merged**